dataprofiler/profilers/float_column_profile.py

```diff
@@ -105,6 +106,25 @@ def __add__(self, other):
 
         return merged_profile
 
+    def diff(self, other_profile, options=None):
+        """
+        Finds the differences for FloatColumnss.
+
+        :param other_profile: profile to find the difference with
+        :type other_profile: FloatColumn
+        :return: the FloatColumn differences
+        :rtype: dict
+        """
+        differences = super().diff(other_profile, options=None)
```

Comment:
I'm adding a general diff in the base column class so I think you will need to be specific that this is the Numerical Stats super.

Reply:
otherwise looks good to me